src/main/java/com/mollin/yapi/YeelightDevice.java

```diff
@@ -36,6 +36,10 @@
      * Device effect duration setting for commands
      */
     private int duration;
+    /**
+     * Additional information about a Yeelight device
+     */
+    private Optional<YeelightDeviceMeta> meta;
```

Comment:
I'm not a huge fan for a "meta" attribute (for example if you instanciate a Bulb with only IP address, there is no "meta" information)